lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToProbabilities.py

```diff
@@ -180,8 +188,9 @@ def test_threshold_above_cube_truncnorm(self):
 
         expected = (np.ones((3, 3, 3)) * [0.8914245, 0.5942867, 0.2971489]).T
         plugin = Plugin(distribution="truncnorm", shape_parameters=[0, np.inf])
-        result = plugin._mean_and_variance_to_probabilities(
-            self.means, self.variances, self.template_cube)
+        result = plugin._location_and_scale_parameters_to_probabilities(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _location_and_scale_parameters_to_probabilities of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4576393)